src/components/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import PropTypes from 'prop-types'
+
+const Button = (props) => {
+  return (
+    <button className={'button'} style={{ backgroundColor: (props.color === 'orange' ? '' : 'lightGray'), width: (props.width === true ? '50%' : '') }}>
+      {props.name}
+    </button >
+  );
+}
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  width: PropTypes.bool.isRequired,
+  color: PropTypes.string
+}
+
```

Comment:
let's add a defaultProps for when the colour is not set. that's where we make it orange by default, we could set a default for wide too